### PR TITLE
Add Changelog for 1.3.0

### PR DESCRIPTION
**Pull request by rgrunber**
Signed-off-by: Roland Grunberg <rgrunber@redhat.com>

- Just waiting for https://github.com/redhat-developer/vscode-java/issues/2225 .

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
 
+## 1.3.0 (January 21st, 2022)
+ * enhancement - Support completion for lambda expressions. See [JLS#1985](https://github.com/eclipse/eclipse.jdt.ls/issues/1985).
+ * enhancement - Add "Convert to Switch Expression" code assist proposal. See [JLS#1935](https://github.com/eclipse/eclipse.jdt.ls/pull/1935).
+ * bug fix - Fix regression in signature help. See [JLS#1980](https://github.com/eclipse/eclipse.jdt.ls/issues/1980).
+ * bug fix - Provide a way to disable embedded JRE. See [#2276](https://github.com/redhat-developer/vscode-java/issues/2276).
+ * bug fix - Projects containing windows symlink directories are not recognized. See [#2264](https://github.com/redhat-developer/vscode-java/issues/2264).
+ * bug fix - JDK installed via Homebrew is not detected. See [#2254](https://github.com/redhat-developer/vscode-java/issues/2254).
+ * bug fix - Avoid duplicate quick fixes when showing all quick fixes on a line. See [#2236](https://github.com/redhat-developer/vscode-java/issues/2236).
+ * bug fix - "Go to References" result contains inacurate references. See [#2227](https://github.com/redhat-developer/vscode-java/issues/2227).
```
Comment (testforstephen) [inline CHANGELOG.md:11]: typos.

inacurate -> inaccurate

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+ * bug fix - Fix gradle wrappers are correctly processed on project import. See [#2218](https://github.com/redhat-developer/vscode-java/issues/2218).
```
Comment (testforstephen) [inline CHANGELOG.md:12]: -> "Fix Gradle wrapper being correctly processed on project import" ?

**Comment by testforstephen** [comment]
Just some minor comments. Once you address them and add #2225 as well, feel free to merge.

File: CHANGELOG.md (hunk 2 above)
Comment (rgrunber) [reply CHANGELOG.md:12]: You're right. It's wrong. I'll reword it.

**Comment by rgrunber** [comment]
Note : I've set the release for the 24th to avoid releasing on a Friday. I plan to have a candidate up by Friday for testing.

**Review by testforstephen — APPROVED** [review]

**Pull request merged**